hotsos/cli.py

```diff
@@ -175,9 +175,11 @@ def main():
     @click.option('--html-escape', default=False, is_flag=True,
                   help=('Apply html escaping to the output so that it is safe '
                         'to display in html.'))
-    @click.option('--format', default='yaml',
-                  help=('Output format. Supported formats are yaml and json. '
-                        'Default is yaml.'))
+    @click.option('--format',
+                  type=click.Choice(['yaml', 'json', 'markdown']),
+                  default='yaml',
+                  show_default=True,
+                  help=('Output format. Supported formats are yaml and json.'))
```

Comment:
and markdown ?

Reply:
Ah yes. I'll address this in the next iteration.